tools/coredump/coredump_test.go

```diff
@@ -11,6 +11,11 @@ import (
 )
 
 func TestCoreDumps(t *testing.T) {
+	var skip = map[string]bool{
+		// https://github.com/open-telemetry/opentelemetry-ebpf-profiler/issues/416
+		"testdata/amd64/alpine320-nobuildid.json": true,
+		"testdata/amd64/alpine320.json":           true,
+	}
```

Comment:
I'd rather add the `skip` field to the `json` file perhaps as a string, where the string is the link to the ticket explaining why it fails.